index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title> my portfolio</title>
+</head>
+
+<body>
+    <main class="container">
+        <section>
+            <a class="logo">logos</a>
```

Comment:
- [ Optional ] Kindly replace the `logos` text with your name or a logo you created for your portfolio because this is your portfolio and it has to convey that message to visitors.
- [ Optional ]  The font size in the design is 18px. Kindly rectify.